### ISSUE TITLE
Cannot build example

### ISSUE THREAD

**Issue by rad-p** [UniversalLogin/UniversalLoginSDK#151]
```
UniversalLoginSDK$ uname -a
Linux hostname 4.4.0-139-generic #165-Ubuntu SMP Wed Oct 24 10:58:50 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
```
```
UniversalLoginSDK$ yarn -v
1.12.3
```
```
UniversalLoginSDK$ node -v
v11.3.0
```
```
UniversalLoginSDK$ yarn
yarn install v1.12.3
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
error https://registry.yarnpkg.com/flatmap-stream/-/flatmap-stream-0.1.2.tgz: Extracting tar content of undefined failed, the file appears to be corrupt: "Unexpected end of data"
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

```

**Comment by FSM1** [comment]
this seems to be caused by the flatmap package being pulled: https://snyk.io/vuln/SNYK-JS-FLATMAPSTREAM-72637

**Comment by FSM1** [comment]
@rad-p removing the dependency from the yarn.lock file fixes the problems. I have submitted a PR to fix this. In the mean time https://github.com/UniversalLogin/UniversalLoginSDK/pull/153.

**Comment by JustynaBroniszewska** [comment]
Hi, this is a good point! This issue was high-priority, so we already resolved it. 
Btw removing `event-stream` from yarn.lock was not enough. We had to update `goeip-lite` to version 1.3.5, that doesn't use `save` dependency, which uses `event-stream`. 

Here is solution: [#154](https://github.com/UniversalLogin/UniversalLoginSDK/pull/154) 

Thank you for your help. We look forward to your next contributions!
